src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -63,9 +63,275 @@ private MobileAdsDataProvider() {
                 "\"s2\":\"article\"",
                 "\"dmn\":\"wikiacom\"",
                 "\"hostpre\":\"",
+                "\"skin\":\"mercury\"",
                 "\"wpage\":\"synthetictests/dfpparams\"",
                 "\"ref\":\"direct\"",
                 "\"lang\":\"en\"",
+                "\"skin\":\"mercury\"",
+                "\"esrb\":\"teen\""
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        }
+    };
+  }
+
+  @DataProvider
+  public static Object[][] dfpParams() {
+    return new Object[][]{
+        {
+            "yugioh",
+            "Dark_Magician",
+            "wka.ent/_yugioh//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"ent\"",
+                "\"s0v\":\"comics\"",
+                "\"s0c\":[\"gaming\",\"anime\",\"videogames\"]",
+                "\"s1\":\"_yugioh\"",
+                "\"s2\":\"article\"",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":",
+                "\"cat\":[",
+                "\"ab\":[",
+                "\"dmn\":\"",
+                "\"skin\":\"mercury\"",
+                "\"ar\":\"4:3\"",
+                "\"artid\":\"278\"",
+                "\"wpage\":\"dark_magician\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"esrb\":[\"everyone\"]",
+                "\"age\":[\"teen\",\"kid\",\"18-34\"]",
+                "\"gnre\":[\"anime\",\"fantasy\",\"strategy\",\"comics\"]",
+                "\"media\":[\"tv\",\"cards\"]",
+                "\"pform\":[\"pc\",\"psp\"]",
+                "\"sex\":[\"m\"]"
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        },
+        {
+            "fallout",
+            "Fallout",
+            "wka.gaming/_fallout//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"gaming\"",
+                "\"s0v\":\"games\"",
+                "\"s0c\":[\"gaming\"]",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"s0c\":[\"gaming\"]" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)